<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sniffer</class>
 <widget class="QMainWindow" name="Sniffer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sniffer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="data.qrc">
    <normaloff>:/images/sniffer_icons_2.png</normaloff>:/images/sniffer_icons_2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"> QMenu::item:selected {
      border-color: darkblue;
      background: rgba(100, 100, 100, 150);
  }
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>185</width>
      <height>297</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QListWidget" name="mListe">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QTextBrowser" name="Logdisplayer">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="devices_comboBox">
       <property name="styleSheet">
        <string notr="true">QComboBox{background-color: rgb(85, 170, 255);}
</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QTableWidget" name="m_tW">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="gridStyle">
        <enum>Qt::DashDotLine</enum>
       </property>
       <property name="rowCount">
        <number>100</number>
       </property>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Packet lenght</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>From</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>To</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Port</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Destination Port</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Protocol</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Info</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Home"/>
    <widget class="QWidget" name="sniffer"/>
    <widget class="QWidget" name="Settings"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>430</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="hbLayout">
     <item>
      <widget class="QTextBrowser" name="homeBrowser">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
        </font>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Comic Sans MS'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/images/sniffer_icons_2.png&quot; /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Comic Sans MS'; font-size:18pt; font-weight:600;&quot;&gt;      S&lt;/span&gt;&lt;span style=&quot; font-family:'Comic Sans MS'; font-size:12pt; font-weight:600;&quot;&gt;niffer 1.0&lt;/span&gt;&lt;span style=&quot; font-family:'Comic Sans MS'; font-size:7.8pt; font-weight:600;&quot;&gt;  &lt;/span&gt;&lt;span style=&quot; font-family:'Comic Sans MS'; font-size:7.8pt;&quot;&gt;              &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Comic Sans MS'; font-size:12pt;&quot;&gt;By Jordan Tchoulayeu&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Comic Sans MS'; font-size:12pt;&quot;&gt;jordanprog@yahoo.fr&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Comic Sans MS'; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Comic Sans MS'; font-size:12pt;&quot;&gt;How to use?&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Comic Sans MS'; font-size:12pt;&quot;&gt;- Go to Sniffer Tab&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Comic Sans MS'; font-size:12pt;&quot;&gt;- Click on find all devices&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Comic Sans MS'; font-size:12pt;&quot;&gt;- Select a device&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Comic Sans MS'; font-size:12pt;&quot;&gt;- Click on lookup device&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Comic Sans MS'; font-size:12pt;&quot;&gt;- Enter the lookup time and wait till it finishes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar{background-color: &quot;lightblue&quot;;}
QToolButton::clicked{
background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionSniffer"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionFindAllDevices"/>
   <addaction name="actionLookupdev"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionFindAllDevices">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mdata.qrc">
     <normaloff>:/images/view-refresh.png</normaloff>:/images/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>findAllDevices</string>
   </property>
  </action>
  <action name="actionLookupdev">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/images/busy.png</normaloff>:/images/busy.png</iconset>
   </property>
   <property name="text">
    <string>LookupDevice</string>
   </property>
  </action>
  <action name="actionFindDevice">
   <property name="text">
    <string>Lookupnet</string>
   </property>
  </action>
  <action name="actionNetLookup">
   <property name="text">
    <string>netLookup</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/images/config.png</normaloff>:/images/config.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/images/home.png</normaloff>:/images/home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionSniffer">
   <property name="icon">
    <iconset resource="mdata.qrc">
     <normaloff>:/images/wifi.png</normaloff>:/images/wifi.png</iconset>
   </property>
   <property name="text">
    <string>Sniffer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="data.qrc"/>
  <include location="mdata.qrc"/>
 </resources>
 <connections/>
</ui>
